tests/RunTests.hs

```diff
@@ -64,7 +64,7 @@ main = hspec $ do
       assertSuccess (bazel ["run", "//tests/multi_repl:c_multi_repl", "--", "-ignore-dot-ghci", "-e", ":load BC.C", "-e", "c"])
 
   describe "failures" $ do
-    all_failure_tests <- bazelQuery "kind(rule, //tests/failures/...) intersect attr('tags', 'manual', //tests/failures/...)"
+    all_failure_tests <- bazelQuery "kind('haskell_library|haskell_binary|haskell_test', //tests/failures/...) intersect attr('tags', 'manual', //tests/failures/...)"
```

Comment:
Why?

Reply:
The `@repl` and `-repl` are now separate targets, so that a query `kind(rule, ...)` returns those as well. But, those targets are not broken, while this test case expects all returned targets to fail. So, we need to be more specific in the query.

Reply:
Ok. If you feel it, please add a comment.

Thank you.

Reply:
Good point, I've added a comment.